manifests/init.pp

```diff
@@ -1,4 +1,155 @@
-# main chrony class
+# @summary Installs and configures chrony
+#
+# @example Install chrony with default options
```

Comment:
I've copied the examples from the README, but not deleted them (yet).  I think it's nice that there are at least a few of the most important examples there and not just in the reference docs.

Reply:
thanks